bigchaindb/commands/bigchaindb.py

```diff
@@ -10,21 +10,16 @@
 import sys
 
 from bigchaindb.common import crypto
-from bigchaindb.common.exceptions import (StartupError,
-                                          DatabaseAlreadyExists,
+from bigchaindb.common.exceptions import (DatabaseAlreadyExists,
                                           KeypairNotFoundException,
                                           DatabaseDoesNotExist)
 import bigchaindb
 from bigchaindb import backend
 from bigchaindb.backend import schema
 from bigchaindb.backend import query
-from bigchaindb.backend.admin import (set_replicas, set_shards, add_replicas,
-                                      remove_replicas)
-from bigchaindb.backend.exceptions import OperationError
 from bigchaindb.commands import utils
 from bigchaindb.commands.messages import (
     CANNOT_START_KEYPAIR_NOT_FOUND,
```

Comment:
I can't imagine why this is still needed. "Keypair not found" is not a reason for error any more, in BigchainDB. There is no keypair (in BigchianDB). (The keypair is in Tendermint but BigchainDB doesn't care about that.)